### PR TITLE
[mio-circle08] Revise to support ext_offset

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise to support ext_offset with
- enable mutable methods with generated schema
- enable to read Buffer offset/size attributes

### CODE REVIEW

**Comment by seanshpark** [comment]
- for #12732
- from #13615

**Review by shs-park — APPROVED** [review]
LGTM!
=)

File: compiler/mio-circle08/src/Reader.cpp [hunk 1]
```diff
@@ -45,6 +46,28 @@ Reader::Reader(const ::circle::Model *model)
   }
 }
 
+Reader::Reader(const ::circle::Model *model, const std::vector<char> *rawdata)
```
Comment (mhs4670go) [inline compiler/mio-circle08/src/Reader.cpp:49]: (optional)
```suggestion
Reader::Reader(const ::circle::Model *model, const std::vector<char> *rawdata): Reader(model)
{
  _rawdata = rawdata;
}
```

**Review by mhs4670go — APPROVED** [review]
LGTM

File: compiler/mio-circle08/src/Reader.cpp (hunk 1 above)
Comment (seanshpark) [reply compiler/mio-circle08/src/Reader.cpp:49]: Ah, I'm planning to remove old `Reader::Reader`

**Pull request merged**